UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -205,6 +205,20 @@ public enum FunctionClassification : ulong
     /// </summary>
     public UndertaleString Name { get; set; }
 
+    /// <summary>
+    /// Different GameMaker release branches. LTS has some but not all features of equivalent newer versions.
+    /// </summary>
+    public enum BranchType
+    {
+        LTS,
+        NonLTS
+    }
+
+    /// <summary>
+    /// The GameMaker release branch of the data file. May be set to "NonLTS" when features exempted from LTS are detected.
+    /// </summary>
+    public BranchType Branch = BranchType.LTS;
```

Comment:
Non-LTS is detected when we find a feature that isn't present on LTS, and there's no point setting the default to "LTS if it's at least the earliest version for which LTS exists" because there are no checks for LTS-ness on older versions.

Reply:
hmm, IMO it just comes off as confusing if i load a game that was made before lts existed and it shows as LTS.